editor/editor_system.cpp

```diff
@@ -26,7 +26,21 @@
 #include "imgui_stdlib.h"
 #include "ImGuizmo.h"
 
-#define DOCUMENTATION_LINK String("https://rootex.readthedocs.io/")
+#define DOCUMENTATION_LINK String("https://rootex.readthedocs.io/")
+
+bool checkReservedName(std::string sceneName)
```

Comment:
This should technically be in the place where entities are created because this check needs to be in both the editor and the game.